src/olympia/amo/tests/test_utils.py

```diff
@@ -254,3 +256,14 @@ def test_adds_etag_header(self):
         etag = '123'
         resp = HttpResponseXSendFile(request=None, path='/', etag=etag)
         assert resp['ETag'] == quote_etag(etag)
+
+
+def test_images_are_small():
+    """A test that will fail if we accidentally include a large image."""
+    large_images = []
+    img_path = os.path.join(settings.ROOT, 'static', 'img')
+    for root, dirs, files in os.walk(img_path):
+        large_images += [
+            os.path.join(root, name) for name in files
+            if os.path.getsize(os.path.join(root, name)) > IMAGE_FILESIZE_MAX]
+    assert not large_images
```

Comment:
Taking "explicit is always better than implicit" into account, I personally would prefer to use `pytest.mark.parametrize` - especially since it'll make finding "where is this image used?"-things easier.

Reply:
gah! I didn't open the issue so didn't see your edit.  Fwiw, using `parametrize` is 10x slower too (10 seconds vs 1 second)